sql/moz-fx-data-shared-prod/org_mozilla_fenix_derived/review_checker_events_v1/query.sql

```diff
@@ -0,0 +1,62 @@
+SELECT
+  DATE(submission_timestamp) AS submission_date,
+  client_info.client_id AS client_id,
+  client_info.os AS os,
+  client_info.os_version AS os_version,
+  MAX(CASE WHEN name = 'address_bar_feature_callout_displayed' THEN 1 ELSE 0 END) is_address_bar_feature_callout_displayed,
+  MAX(CASE WHEN name = 'address_bar_icon_clicked' THEN 1 ELSE 0 END) is_address_bar_icon_clicked,
+  MAX(CASE WHEN name = 'address_bar_icon_displayed' THEN 1 ELSE 0 END) is_address_bar_icon_displayed,
+  MAX(CASE WHEN name = 'product_page_visits' THEN 1 ELSE 0 END) is_product_page_visits,
+  MAX(CASE WHEN name = 'surface_analyze_reviews_none_available_clicked' THEN 1 ELSE 0 END) is_surface_analyze_reviews_none_available_clicked,
+  MAX(CASE WHEN name = 'surface_closed' THEN 1 ELSE 0 END) is_surface_closed,
+  MAX(CASE WHEN name = 'surface_displayed' THEN 1 ELSE 0 END) is_surface_displayed,
+  MAX(CASE WHEN name = 'surface_expand_settings' THEN 1 ELSE 0 END) is_surface_settings_expand_clicked,
```

Comment:
surface_expand_settings is correct for fenix.  do we want to match the alias to that text or do we want to match it to the canonical.  i'm thinking match to what fenix telemetry (is_surface_expand_settings) and then on the back end in looker I can make them all the same as desktop if we want.